app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -323,6 +323,13 @@ protected void onCreate(Bundle savedInstanceState) {
         mPoorPerformanceAllowList = new HashSet<>();
         checkForCrash();
 
+        // Show the launch dialogs, if needed.
+        if (!showTermsServiceDialogIfNeeded()) {
+            if (!showPrivacyDialogIfNeeded()) {
+                showWhatsNewDialogIfNeeded();
+            }
+        }
+
```

Comment:
Seems correct, but I still get the whats new dialog in the 2nd launch of wolvic using the China release